phpstan.neon.dist

```diff
@@ -0,0 +1,39 @@
+parameters:
+    level: 3
+    paths:
+        - %currentWorkingDirectory%/lib
+    autoload_files:
+        - %currentWorkingDirectory%/tests/phpstan-polyfill.php
+    reportUnmatchedIgnoredErrors: false
+    ignoreErrors:
+        # extension not available
+        - '~^(Used )?(Function|Constant) sasql_\S+ not found\.\z~i'
```

Comment:
Could we generate a stub in the longer term? As long as we officially support the driver.

Reply:
I have no idea. PhpStorm stubs don't have sasql and I didn't find any documentation either, nor PECL extension. Working with SAP SQL Anywhere is really hard. :(

Reply:
For future reference, I found a few working links at https://archive.sap.com/documents/docs/DOC-40537. I'll try to drop in the Windows DLL into PHP 5 and see if I can dump the signatures. Sometime.

Reply:
The link lead me to: http://dcx.sap.com/index.html#sa160/en/dbprogramming/php-api.html
This looks good enough to build the stubs. Maybe i.e. @carusogabriel may want to help here? :)

But we should probably consider dropping the platform/driver altogether.
Given that DBAL 3 will require PHP 7.2, and SQL Anywhere is barely keeping up (the link only lists PHP 5 modules)...

Reply:
Nevermind, did this during Slack outage. :stuck_out_tongue: 
https://github.com/JetBrains/phpstorm-stubs/pull/382